Makefile

```diff
@@ -231,3 +233,31 @@ catalog-build: opm ## Build a catalog image.
 .PHONY: catalog-push
 catalog-push: ## Push a catalog image.
 	$(MAKE) docker-push IMG=$(CATALOG_IMG)
+
```

Comment:
this chunk should go away when you rebase against master

Reply:
> this chunk should go away when you rebase against master

Yes but there is more to it. See below.